packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -56,8 +58,7 @@ export const MenuList = compose<MenuListType>({
               <Slots.focusZone
                 componentRef={focusZoneRef}
                 focusZoneDirection={'vertical'}
-                defaultTabbableElement={focusZoneRef}
-                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                defaultTabbableElement={context.isSubmenu ? null : focusZoneRef} // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Could we add a comment saying why setting `defaultTabbableElement` affects which menu item gets focused?